### DISCUSSION TITLE
[Kavita] Custom Favicon

### DISCUSSION THREAD

**Discussion by hillmanation** [Kareadita/Kavita#3275] [Ideas]
### Idea Description

Essentially add the ability to upload and use a custom favicon and main icon for the server. I've tested this by replacing the applicable files manually and it seems to work fine. I will likely spend some of this next week or so working on a pull request for this but if this is already a planned feature or there is a specific settings page I should look at adding it to I'm all ears.

### Idea Category

User Experience

### Duration of Using Kavita

2 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by hillmanation** [comment]
I think it would make sense too for this to be a Kavita+ unlocked feature, if implemented.

**Comment by majora2007** [comment]
You can just override the favicon on your install manually. I'm not interested in providing a mechanism to rebrand Kavita.
